render/auto.go

```diff
@@ -113,34 +114,38 @@ func (ir htmlAutoRenderer) Render(w io.Writer, data Data) error {
 				return err
 			}
 			if data["method"] == "PUT" {
-				return ir.HTML(fmt.Sprintf("%s/edit.html", templatePrefix)).Render(w, data)
+				return ir.osIndependentHTML(fmt.Sprintf("%s/edit.html", templatePrefix)).Render(w, data)
 			}
-			return ir.HTML(fmt.Sprintf("%s/new.html", templatePrefix)).Render(w, data)
+			return ir.osIndependentHTML(fmt.Sprintf("%s/new.html", templatePrefix)).Render(w, data)
 		}
 		return nil
 	}
 	cp, ok := data["current_path"].(string)
 
 	defCase := func() error {
-		return ir.HTML(fmt.Sprintf("%s/%s.html", templatePrefix, "index")).Render(w, data)
+		return ir.osIndependentHTML(fmt.Sprintf("%s/%s.html", templatePrefix, "index")).Render(w, data)
```

Comment:
@daHaimi could we change this to be 
```suggestion
		return ir.HTML(filepath.Join(tremplatePrefix, "index.html").Render(w, data)
```

And the same across this file?